### PR TITLE
Front page

### PR DESCRIPTION
**Pull request by Carolien989**

### CODE REVIEW

File: Switchboard/public/app/components/frontPage/FrontPage.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+#Frontpage .pictures{
+    -webkit-align-content: center;
+    align-content: center;
+    padding-left: 100px;
+    padding-right: 100px;
+    padding-top: 100px;
+}
+}
```
Comment (killje) [inline Switchboard/public/app/components/frontPage/FrontPage.css]: a } to many

File: Switchboard/public/app/components/frontPage/FrontPage.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#Frontpage .leftplacement{
+    float:left;
```
Comment (killje) [inline Switchboard/public/app/components/frontPage/FrontPage.css]: No floats or this happens 
brokenfloat

Comment (killje) [inline Switchboard/public/app/components/frontPage/FrontPage.css]: Use:
display: inline-block;
width: 50%
margin-right: -2px; /* and margin-left for right placement */

File: Switchboard/public/app/components/frontPage/FrontPage.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+#Frontpage .rightplacement{
+    float:right;
+}
+
+#Frontpage .resize {
+    width: 150px;
+    height: auto;
+}
+
+
```
Comment (killje) [inline Switchboard/public/app/components/frontPage/FrontPage.css]: Add text-align left for widths smaller as 400px as its better to read:

@media ( max-width:400px ) {
    #Frontpage .info {
        text-align: left;   
    }
}

File: Switchboard/public/app/components/frontPage/FrontPageView.html [hunk 4]
```diff
@@ -1,2 +1,19 @@
...
+
+        <h2 class="Header">Welcome to the Universal API Switchboard!</h2>
+        <h4>This application is build to let you manage API connections between different applications.</h4>
+        <h4>This switchboard is made as a university project for the 
+            course “Software Engineering” of the University of Groningen commissioned by culture centre “De Klinker”.</h4>
+    </div> 
+
+    <div class="pictures">
+        <a class='leftplacement' href="https://www.indeklinker.nl/"><img class='resize' src="image/klinker.png"></a>
+        <a class='rightplacement'><img class="resize" src="image/rug.png"></a>
```
Comment (killje) [inline Switchboard/public/app/components/frontPage/FrontPageView.html]: Add rug homepage here?

**Comment by killje** [comment]
The second outdated comment is not yet fixed.

**Comment by bbusagroningen** [comment]
It looks pretty decent, I like it!

**Pull request merged**